### PR TITLE
[CU-86bymbvd4] Map UI Sidebar

### PR DESCRIPTION
**Pull request by BejenaruIoanMatei**
Sidebarul contine butoanele (fara actiuni atribuite fiecarui buton in parte).

### CODE REVIEW

**Comment by sergane13** [comment]
Task linked: [CU-86bymbvd4 [FE-008] Map UI Sidebar](https://app.clickup.com/t/86bymbvd4)

File: package.json [hunk 1]
```diff
@@ -33,4 +33,5 @@
     "eslint-plugin-react-refresh": "^0.4.6",
     "vite": "^5.2.0"
   }
-}
+
```
Comment (sergane13) [inline package.json:36]: blank space. Please remove.

File: src/shared/App.jsx
Comment (sergane13) [inline src/shared/App.jsx:1]: Please add the sidebar together with components made by Gabriela.

**Review by sergane13 — CHANGES_REQUESTED** [review]
Please do the changes and get rid of merge conflicts erros.

**Review by BogdanMihaica — APPROVED** [review]

**Review by sergane13 — APPROVED** [review]
Looks fine to me.

**Pull request merged**